internal/cmd/io_binary.go

```diff
@@ -105,6 +107,34 @@ func NewBinaryIO(ctx context.Context, id string, uri *url.URL) (_ UpstreamIO, er
 	}, nil
 }
 
+// sanitizePath parses the URL object and returns an absolute path to the logging driver
+func sanitizePath(uri *url.URL) (sanitized string) {
+	path := uri.Path
+	// Convert windows-style slashes to linux-style to clean the path
+	path = strings.Replace(path, "\\", "/", -1)
+	path = filepath.Clean(path)
+	// Path is clean, convert back
+	sanitized = strings.Replace(path, "/", "\\", -1)
+	if sanitized == "." {
+		sanitized = ""
+	}
+
+	// Strip any leading '\\' if the path contains a hard drive in it.
+	if strings.Contains(sanitized, ":\\") {
```

Comment:
I'm confused about this section. Could `:\\` be anywhere in the path?

Reply:
like if we have multiple volumes, e.g. `C:\path\to\something` and `D:\path\to\something`. or am I missing something?